### PR TITLE
Feature/core 549/core 638/cover common configs changes for content blocks

### PR DESCRIPTION
**Pull request by pascaleli**

### CODE REVIEW

File: cypress/e2e/undo-redo/block-configs-should-update-flow-state/location-block-configs.cy.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+expect(values).to.have.length(3)
+
+// the order follows how we set the flow modes from createFlow() command
+expect(values[0].value).to.equal(blockConfigs.resources.sms)
+expect(values[1].value).to.equal(blockConfigs.resources.ussd)
+expect(values[2].value).to.contains('.mp3')
+})
+
+//##### threshold & validating if the values are saved properly#####
+    cy.get('[data-cy="accuracy-threshold"]')
```
Comment (pascaleli) [inline cypress/e2e/undo-redo/block-configs-should-update-flow-state/location-block-configs.cy.js:71]: Hi @safydy-r 
Please i could not hit this selector cy.get('[data-cy="accuracy-threshold"]') in the location script eventhough I added it to the .vue

File: cypress/e2e/undo-redo/block-configs-should-update-flow-state/location-block-configs.cy.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    .type(blockConfigs.threshold.accuracy_threshold)
+    firstBlockState().its('config.accuracy_threshold_meters').should('equal', 5+blockConfigs.threshold.accuracy_threshold)
+
+    cy.get('[data-cy="accuracy_timeout"]')
```
Comment (pascaleli) [inline cypress/e2e/undo-redo/block-configs-should-update-flow-state/location-block-configs.cy.js:75]: This as well @safydy-r

File: src/components/interaction-designer/block-editors/ThresholdEditor.vue [hunk 3]
```diff
@@ -8,7 +8,8 @@
           :regex-float-filtering="'[0-9.,]'"
           :label="'flow-builder.accuracy-threshold-in-meters' | trans"
           :placeholder="'flow-builder.enter-value' | trans"
-          :valid-state="isValid" />
+          :valid-state="isValid" 
+          :data-cy="'accuracy-threshold'"/>
```
Comment (pascaleli) [inline src/components/interaction-designer/block-editors/ThresholdEditor.vue:12]: I have added them here , but still could not hit

File: cypress/e2e/undo-redo/block-configs-should-update-flow-state/location-block-configs.cy.js (hunk 1 above)
Comment (safydy-r) [reply cypress/e2e/undo-redo/block-configs-should-update-flow-state/location-block-configs.cy.js:71]: hey Pascal, I will resume this

**Review by safydy-r — COMMENTED** [review]
Thanks of this test @pascaleli , it's a good start. Nice job here.

I have few remark to improve the future PRs though:
- use the FLOIP repo directly, not the forked one to create PRs. That way, your colleague could resume the work easily or update your branch smoothly
- for data-cy attribute, we should talk next week about some convention. I'd suggest we add prefixes about the nature of the element. eg: `max-digits--input` instead of `max-digits`, `keypad-digits--selector` instead of `keypad-digits`
- use indentations to make the code easier to read, eg on these lines https://github.com/FLOIP/flow-builder/pull/497/files#diff-eddb2830d81dc84a9dda8d6611b7512df27254fbcbf87bb74e28e1fc06e1fd82R32

File: cypress/e2e/undo-redo/block-configs-should-update-flow-state/location-block-configs.cy.js (hunk 1 above)
Comment (safydy-r) [reply cypress/e2e/undo-redo/block-configs-should-update-flow-state/location-block-configs.cy.js:71]: @pascaleli , you need to find the appropriate input or textarea first. See my change here https://github.com/FLOIP/flow-builder/pull/499/files#diff-dd9459d1e5ea3e9abde204db168933e5f8e06f467b7b9cfa4b99948b72ddeae1R29

**Pull request merged**